sandbox

```diff
@@ -352,8 +368,58 @@ sandbox () {
       fi
     done
 
-    overwrite "* Account processing complete."
+    overwrite "* Account download complete."
+    # Newline for the progress bar to use.
+    echo ""
+
+    DONE=false
+    STARTED=false
+    while [ $DONE == "false" ]; do
+      sleep 0.1
+
+      local RES
+      local TOTAL
+      local PROGRESS
+
+      RES="$(status_helper)"
+      STARTED=true
+      TOTAL=$(echo $RES | grep -o 'Catchpoint total KVs: [[:digit:]]*' | cut -d':' -f 2 )
+      PROGRESS=$(echo $RES | grep -o 'Catchpoint KVs processed: [[:digit:]]*' | cut -d':' -f 2 )
+
+      if [ $TOTAL == $PROGRESS ]; then
+        DONE=true
+      else
+        progress_bar "Processing boxes:" "$PROGRESS" "$TOTAL"
```

Comment:
Neat that we can do this!

Reply:
this and the spinner are absurdly complicated for what they are but make sandbox feel so much more polished.

Reply:
In addition, it's cool we have the boxes information to report.